db/seeds.rb

```diff
@@ -8,3 +9,10 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+User.create(
+  name: ENV.fetch('DEFAULT_USER_NAME', nil),
+  email: ENV.fetch('DEFAULT_USER_EMAIL', nil),
+  password: ENV.fetch('DEFAULT_USER_PASSWORD', nil),
+  password_confirmation: ENV.fetch('DEFAULT_USER_PASSWORD', nil)
+)
```

Comment:
You can move everything from `League#seed` into this file, then you don't need to hit that route to see the db. You can just run `rails db:seed` from the command line and it will seed, along with the user data.

Reply:
The problem I have in "production" is that at the free tier Render does not allow me Console access.

What I am thinking is to make the seed idempotent for it to run as part of the deployment process. 

But I also want to look closer into maintenance tasks. 

The future of League is to have its own model. It will be interesting to add all the migrations to add a parent entity.

Reply:
I changed to create only if it does not exist and to run `db:seed` at Render.

Now I am wondering about: (i) how to do authorization? I would like to have an admin role; (ii) how do I get to get and add attributes to the session;

Reply:
For authorization I recommend you use [Pundit](https://github.com/varvet/pundit).

You may also want to look into [CanCanCan](https://github.com/CanCanCommunity/cancancan).

Both are used at Shopify. Pundit is more popular these days because, imo, it's better. CanCanCan used to be the most popular gem for authorization, but now it's mostly used in legacy apps.

Reply:
> how do I get to get and add attributes to the session

What do you mean? What are you trying to do? Store some data in the session, like who is authenticated?

Reply:
Created #29 for authorization

Reply:
Yes, the user ID for instance. I just found out that `Devise` has some helper functions and ones is `current_user`. That should do.